dist/css/demo.rtl.css

```diff
@@ -244,76 +331,107 @@ pre.highlight,
   right: 0;
   top: 0;
   height: 100%;
-  border-radius: 2px 2px 0 0; }
+  border-radius: 2px 2px 0 0;
+}
 
 @media not print {
   .theme-dark .example {
     background-color: #1b2434;
-    border-color: #2c3c56; }
+    border-color: #2c3c56;
+  }
   .theme-dark .example-content {
-    color: #f8fafc; }
+    color: #f8fafc;
+  }
   .theme-dark .example-code {
     border-color: #2c3c56;
-    border-top: none; } }
-
+    border-top: none;
+  }
+}
+@media not print {
+  @media (prefers-color-scheme: dark) {
+    .theme-dark-auto .example {
+      background-color: #1b2434;
+      border-color: #2c3c56;
+    }
+    .theme-dark-auto .example-content {
+      color: #f8fafc;
+    }
+    .theme-dark-auto .example-code {
+      border-color: #2c3c56;
+      border-top: none;
+    }
+  }
+}
```

Comment:
and this is the rule that was missing before? the fix seems to work

Reply:
yes, it fixes `theme-dark-auto` class